examples/openvino/audio-classification/README.md

```diff
@@ -0,0 +1,104 @@
+<!---
+Copyright 2021 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Audio classification examples
+
+The following examples showcase how to fine-tune `Wav2Vec2` for audio classification using PyTorch.
```

Comment:
While it is nice to show that NNCF finetuning is just adding a few commands to the PyTorch example, it is a bit confusing to me to show how to finetune with PyTorch in the OpenVINO example. Could we refer with a link to the PyTorch example and in the OpenVINO examples just show OpenVINO/NNCF code?

Reply:
@helena-intel I am not following, could you elaborate what you are looking for? what do you mean by OpenVINO example?

Reply:
The other examples show basic QAT with NNCF first, and then JPQD. For example: "Here we show how to apply quantization aware training (QAT) on a fine-tuned Vision Transformer (ViT) on the beans dataset (to classify the disease type of bean leaves)". This example says  "how to fine-tune `Wav2Vec2` for audio classification using PyTorch.". For consistency and clarity, it is nice if this example also mentions NNCF/QAT for the first example, similar to the other examples.

In general for the examples, as a user I would like to know when to use the first example and when the second. If JQPD is better, should I just start with that? Is there any benefit still for just quantization?

We should preferably not just list examples as "how" to do things but also give a bit of a context about why to choose a particular method.

Reply:
@helena-intel, do you mean that we should have a description of how to quantize the model in every example prior to deeper optimization methods such as JPQD? It makes sense to me if we have such consistency across all the samples but it requires some extra work to create a config and verify that it works. 
@vuiseng9, is it possible for you to check if QAT (with no pruning and distillation) works and add examples of the commands in every document?

Reply:
@AlexKoff88 Yes, we did that. That was how we found out that the default QAT doesn't work for wav2vec2 and provided a tailored config and relative performance improvement from JPQD in the documentation was compared against the QAT.

In all the support tasks, there are quantization and JPQD samples.